### DISCUSSION TITLE
The `replace` option in python SDK's LiveOptions is not working for punctuation characters

### DISCUSSION THREAD

**Discussion by rpriyanshu9** [deepgram/community#853] [General help] [answered]
### Which Deepgram product feature are you using?

Deepgram SDKs

### Details

When I'm initialising the Deepgram client, I'm passing in a valid Find and Replace parameter to the LiveOptions:
```py
replace=["comma:,","period:.","colon::"]
```
This doesn't work in the transcribed text I get back. Instead if I update the F&R options like:
```py
replace=["comma:cat","period:dog","colon:rat"]
```
It starts working as expected.

### If you are making a request to the Deepgram API and have a request ID, please paste it below:

727c0fd0-521e-49bc-acae-94404a9245bd

**Comment by team-deepgram** [comment]
Thanks for asking your question about Deepgram! If you didn't already include it in your post, please be sure to add as much detail as possible so we can assist you efficiently, such as:
- The `request_id` if you have a question about your requests or transcription responses.
- The features you used or the full `api.deepgram.com` URL you sent your request to, including parameters.
- Any code snippets you can share.

**Comment by jpvajda** [comment]
@rpriyanshu9  As far as I know find and replace only works on phrases or words, not punctuation characters right now. This might explain why you are seeing behavior. 

What is it you are attempting to do here? 

Our docs explain this feature here: 
https://developers.deepgram.com/docs/find-and-replace

**Reply by rpriyanshu9** [reply]
@jpvajda I'm trying to replace a dictation tool used by medical clinicians. For the dictation to work, I was trying to use the find & replace settings to replace punctuation characters. The commands provided by the `dictation` setting of Deepgram are very limited.

**Reply by jpvajda** [reply, marked as answer]
@rpriyanshu9  To enable Dictation, set the dictation parameter to true in your API request. Note that the punctuate parameter must also be enabled for Dictation to work.

https://developers.deepgram.com/docs/dictation

Though I did some testing of this feature and it wasn't able to pick up commas or periods very well.

We've haven't actually improved dictation in a while, but I'll share this with our product team so they aware of the limits you are seeing.
